packages/firefuel/lib/src/firefuel_collection.dart

```diff
@@ -150,12 +147,12 @@ abstract class FirefuelCollection<T extends Serializable>
   }
 
   @override
-  Future<Null> updateOrCreate({
+  Future<T> updateOrCreate({
     required DocumentId docId,
     required T value,
   }) async {
     await ref.doc(docId.docId).set(value, SetOptions(merge: true));
 
-    return null;
+    return (await read(docId))!;
   }
 }
```

Comment:
It is necessary to perform a read here?

If we are accepting the value & the id, why do we need to return it via a read?

Reply:
Because the signature was change to return a `T` and we don't know how to convert the docId and value into a `T` since they do that with the `FromFirestore` method

Reply:
Well, I guess they are passing in the T, so...you're right. I can update this.